docs/glossary.md

```diff
@@ -0,0 +1,62 @@
+# OLM Glossary
+
+A list of OLM terms, with definitions and common acronyms.
+
+## CustomResourceDefinitions
+
+### CatalogSources
+
+**Definition**: The CatalogSource is a repository of [CSVs](#ClusterServiceVersion), [CRDs](#CustomResourceDefinitions), and packages that define an application. 
+**Acronyms**: CatSrc(s)
+
+### ClusterServiceVersion
+
+**Definition**: The ClusterServiceVersion is a required resource for an Operator which includes application metadata such as name, description, version, repository link, labels, icon, etc. It also includes a list of corresponding/required resources and their statuses, install strategy of the Operator as [deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), supported install modes, etc.
+**Acronyms**: CSV(s)
+
+### InstallPlans
+
+**Definition**: The InstallPlan lists out all resources to be created in order to automatically install or upgrade a [CSV](#ClusterServiceVersion). 
+**Acronyms**: IP(s)
+
+### OperatorGroups
+
+**Definition**: The operator group selects a set of target namespaces in which to generate required RBAC access for its member Operators.
+**Acronyms**: OG(s)
+
+### OperatorSources
+
+**Definition**: The operator source defines the external data store used to store Operator bundles. It contains information about the catalog of operators including its name, type, namespace, and display name. 
+**Acronyms**: OpSrc(s)
+
+### Subscriptions
+
+**Definition**: The subscription defines the channel and the source of Operator updates and is used to keep the CSV updated by tracking a channel in a package. 
+**Acronyms**: Subs(s)
+
+## OLM Concepts
+
+### Bundle
+
+**Definition**: A collection of Operator manifests and metadata which together form a unique version of an Operator that can be installed onto the cluster. 
+**Acronyms**: Package
+
+### Catalog Image
+
+**Definition**: A catalog image is an image that contains Operator bundles that are stored in a database. These are then exposed over gRPC to the OLM Operator to allow users to query and install new Operators onto the cluster. 
```

Comment:
It's not clear that this is the same thing as an index

Reply:
Do you mean `OPM index`? And is adding a parenthesis name helping?

Reply:
This may make a great entry for an alias.

Reply:
Should we mention `gRPC` here? That feels like an implementation detail. The important part is that the Index is a containerized datastore that describes a set of operator and update metadata that can be installed onto a cluster via OLM.

Reply:
> This may make a great entry for an alias.

This has been a bit back and forth. It was part of the alias before Evan suggested not clear. I'll keep them as an alias again. I think it makes sense and I might have misunderstood that first comment.